sql/upgrade/1.3.3_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -87,3 +87,9 @@ delimiter ;
 CALL ct_dolphin_T_t_ds_process_definition_version;
 DROP PROCEDURE ct_dolphin_T_t_ds_process_definition_version;
 
+
+-- add t_ds_resources_un
+ALTER TABLE t_ds_resources ADD CONSTRAINT t_ds_resources_un UNIQUE KEY (full_name,`type`);
```

Comment:
Hi,This place need judge whether t_ds_resources_un exist,you can use the process function like the above uc_dolphin_T_t_ds_task_instance_A_first_submit_time